tests/Application/templates/bundles/SyliusAdminBundle/Product/Tab/_details.html.twig

```diff
@@ -23,8 +23,9 @@
                     <div class="ui hidden element">
                         {% if product.simple %}
                             {{ form_row(form.variant.translations) }}
+                            {{ form_row(form.variantSelectionMethod) }}
                         {% endif %}
-                        {{ form_row(form.variantSelectionMethod) }}
+
```

Comment:
Why this code was moved? I don't understand business logic there.

Reply:
```
{% if product.simple %}
    {{ form_row(form.variant.translations) }}
    {{ form_row(form.variantSelectionMethod) }}
{% endif %}
```
This code was by default in AdminBundle/Resources/views/Product/Tab/_details.html.twig. Leaving `{{ form_row(form.variantSelectionMethod) }}` outside "if" caused an error "Field 'variantSelectionMethod' has already been rendered" in product with variants.